common/v2/services/EthService/ens/__tests__/validators.spec.ts

```diff
@@ -4,9 +4,7 @@ describe('isValidENSName', () => {
   const valid = ['bob.eth', 'edouard.eth'];
 
   const invalid = [
-    'et.eth', // second level domain must be longer than 2
```

Comment:
Looking at https://opensea.io/ens it seems that 2 char ens names are still invalid.
Why do we remove this test?

Reply:
See discussion here: https://mycryptohq.slack.com/archives/G8G1S4TG9/p1581453596290900

If you feel the other way around, I'm open to changing it.

Reply:
I suggested the edit here. The restrictions (in this case, 2 char) should only be enforced on the UI when registering a name - not on readonly lookup, as we should rely on the onchain registery response instead of implementing our own rulesets